### PR TITLE
feat: remove container start command 

### PR DESCRIPTION
**Pull request by gbockus-sf**
### What does this PR do?
We are removing this command. Users are to use the local version instead.

### What issues does this PR fix or reference?
[@W-12553533@](https://gus.lightning.force.com/a07EE00001LMfHyYAL) / @W-12345678@

https://github.com/salesforcecli/plugin-functions/pull/612
https://github.com/heroku/sf-functions-core/pull/231

### Functionality Before
Users could start a function using Docker.

### Functionality After
Users will no longer be able to start a function using Docker.

Branch created off https://github.com/forcedotcom/salesforcedx-vscode/pull/4687 with added int test fixes.

### CODE REVIEW

**Review by gbockus-sf — COMMENTED** [review]
QE for feat: remove container start command

* Javascript 
    * Start ✅
    * Invoke✅
    * Debug ✅
    * Stop ✅
* Java
    * Start ✅
    * Invoke✅
    * Debug ✅
    * Stop ✅
* Typescript
    * Start ✅
        * Fails the first time with SFDX: Start Function exited unexpectedly
            Could not execute npm build script: Error: Command failed with exit code 1: npm run build —prefix /Users/gbockus/Desktop/TestWithVSixClassCreate/functions/typetesttwo
    * Invoke✅
    * Debug ❌ - verified this is the same as current prod behavior
    * Stop ✅
* Verify no container options are present in command Palette ✅

NOTES: 

* The old version of the javascript function template no longer works. Just silently fails to start

File: packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/forceFunctionStop.test.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import { workspaceUtils } from '../../../../src/util';
 
 describe('Force Function Stop Integration Tests', () => {
   let sandbox: SinonSandbox;
-  const functionsBinaryStub: {
+  const functionsProcessInstStub: {
```
Comment (gbockus-sf) [inline packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/forceFunctionStop.test.ts:22]: updated to use containerless flow

**Comment by saracope** [comment]
@gbockus-sf I was going to put in a PR to bump @heroku/functions-core to the latest version ([v0.7.0](https://github.com/heroku/sf-functions-core/releases/tag/v0.7.0)) which includes a related to change to remove the Docker container functionality ([#231](https://github.com/heroku/sf-functions-core/pull/231)).  

Do you mind adding that version bump here or would you prefer I do that PR separately.

**Review by randi274 — APPROVED** [review]

**Comment by gbockus-sf** [comment]
Hi @saracope Please open a separate PR if you don't mind.  I took a look at the changes from 0.5.1 -> 0.7.0 and seems like it should get it's own round of QE.

**Review by sbudhirajadoc — CHANGES_REQUESTED** [review]
Text missing from Invoke Function section. Also "Stop Function" header is now missing. See https://developer.salesforce.com/tools/vscode/en/functions/overview

**Review by sbudhirajadoc — APPROVED** [review]

**Pull request merged**